lightwood/mixers/sklearn_mixer.py

```diff
@@ -0,0 +1,185 @@
+                weight_map = self.targets[target_col_name]['weights']
+                if weight_map is None:
+                    sample_weight = [1] * len(Y_train)
+                else:
+                    sample_weight = [weight_map[str(val)] for val in Y_train]
+
+                model_classes_and_accuracies = []
+                for model_class, model_kwargs in CLASSIFICATION_MODELS:
+                    model = model_class(**model_kwargs)
+                    model.fit(X_train, Y_train, sample_weight=sample_weight)
+                    accuracy = balanced_accuracy_score(
+                        Y_test,
+                        model.predict(X_test)
+                    )
+
+                    model_classes_and_accuracies.append((
+                        (model_class, model_kwargs),
+                        accuracy
+                    ))
+                
+                (best_model_class, best_model_kwargs), _ = max(
+                    model_classes_and_accuracies,
+                    key=lambda x: x[-1]
+                )
+
+                self.targets[target_col_name]['model'] = best_model_class(**best_model_kwargs)
+
+            elif self.targets[target_col_name]['type'] == COLUMN_DATA_TYPES.NUMERIC:
+                model_classes_and_accuracies = []
+                for model_class, model_kwargs in REGRESSION_MODELS:
+                    model = model_class(**model_kwargs)
+                    model.fit(X_train, Y_train)
+                    accuracy = r2_score(Y_test, model.predict(X_test))
+
+                    log.info('[MODEL/COLUMN/TEST_ACCURACY]: {}/{}/{}'.format(
+                        model_class.__name__,
+                        target_col_name,
+                        accuracy
+                    ))
+
+                    model_classes_and_accuracies.append((
+                        (model_class, model_kwargs),
+                        accuracy
+                    ))
+                
+                (best_model_class, best_model_kwargs), best_accuracy = max(
+                    model_classes_and_accuracies,
+                    key=lambda x: x[-1]
+                )
+
+                self.targets[target_col_name]['model'] = best_model_class(**best_model_kwargs)
+
+                log.info('Selecting {} for column {} (accuracy={})'.format(
+                    best_model_class.__name__,
+                    target_col_name,
+                    best_accuracy
+                ))
+
+                if self.quantiles is not None:
+                    self.targets[target_col_name]['quantile_models'] = {}
+                    for i, quantile in enumerate(self.quantiles):
+                        self.targets[target_col_name]['quantile_models'][i] = best_model_class(**best_model_kwargs)
+
+            else:
+                self.targets[target_col_name]['model'] = None
+        
+        # Fit best model of each column on [train_ds + test_ds]
```

Comment:
I was expecting I'd have to make a comment reminding you to do this bit, I am oh so very happy this is already here ^^